### ISSUE TITLE
add proper exports, check with -fno-common -fvisibility-hidden

### ISSUE THREAD

**Issue by rurban** [rurban/dieharder#1]
Current;y it only works on linux.
For windows or newer compilers I had to add `-fcommon -fvisibility-default`

**Comment by marcusmueller** [comment]
Yes, this needs to be implemented for modern compilability. Thanks @rurban.
